frontend/src/components/pages/TranslationPage.tsx

```diff
@@ -119,12 +122,15 @@ const TranslationPage = () => {
   const clearUnsavedChangesMap = () => {
     setChangedStoryLines(new Map());
   };
+  const tooltipCopy =
```

Comment:
Since we reuse `toolTipCopy` - we should store it in a `utils/Copy.ts` file as a const (something like `TRANSLATION_PAGE_TOOL_TIP_COPY`) and import it across these files.